python/src/aiconfig/default_parsers/openai.py

```diff
@@ -156,6 +156,25 @@ async def serialize(
                     outputs=assistant_output,
                 )
                 prompts.append(prompt)
+            else:
+                # If the first message is an assistant message, create an empty prompt for it.
+                if i == 0:
+                    # Pull assistant response
+                    assistant_output = build_output_data(conversation_data["messages"][i])
+                    # Build prompt with empty input, output as the assistant response
+                    prompt = Prompt(
+                        name=new_prompt_name,
+                        input="",
```

Comment:
Can this be set to null instead of empty string?

Reply:
our schema doesn't allow inputs to be None/null type

Reply:
Would it be a valid alternative to use the PromptInput type for the input and just have the data be the assistant message?

Reply:
I think my answer to that is no.

The PromptInput type is geared towards input, so storing this as an input would either have the implications that this is now an "input", or, we would have to define some additional logic to parse this into the "assistant message"

Some additional context to this it that the openai model parser does support storing the messages array itself as an input to the prompt; but that is a little hacky